rollup.config.js

```diff
@@ -0,0 +1,183 @@
+import babel from 'rollup-plugin-babel'
+import json from 'rollup-plugin-json'
+import resolve from 'rollup-plugin-node-resolve'
+import cssOnly from 'rollup-plugin-css-only'
+import replace from 'rollup-plugin-replace'
+import { terser } from 'rollup-plugin-terser'
+import pkg from './package.json'
+
```

Comment:
```suggestion
const process = require('process');
```

Reply:
@atomiks I believe this might fix the issue where `process` is undefined for me

Reply:
@atomiks would you be open to trying this?

Reply:
See my comment in #530.  Also, I'm pretty sure requiring the module like that just polyfills the environment during rollup compile time, and doesn't do anything to the dist files that get bundled.

Reply:
I believe you are correct. I just tried forking and linking and it didn't help.